autoscheduler/frontend/src/components/SchedulingPage/SchedulingPage.tsx

```diff
@@ -48,6 +57,23 @@ const SchedulingPage: React.FC<SchedulingPageProps> = ({
       </div>
       <div className={styles.scheduleContainer}>
         <Schedule />
+        <div
+          className={styles.fullscreenButtonContainer}
+          style={{ backgroundColor: theme.palette.primary.main }}
+        >
+          <ThemeProvider theme={whiteButtonTheme}>
+            <Typography color="primary" className={styles.totalHoursText}>
+              Total Hours: 15
```

Comment:
Might want to dummy this out before merging, but leave the structure so it can be readded later with minimal effort

Reply:
What did you have in mind for dummying this out? I was just going to leave this here and set the color to be transparent (`display: none` makes the fullscreen button shift left into the total hours text space b/c space-between)